redis/connection.py

```diff
@@ -715,7 +715,7 @@ class UnixDomainSocketConnection(Connection):
     def __init__(self, path='', db=0, password=None,
                  socket_timeout=None, encoding='utf-8',
                  encoding_errors='strict', decode_responses=False,
-                 retry_on_timeout=False,
+                 retry_on_timeout=False, socket_connect_timeout=None,
```

Comment:
Oh, I see. I completely missed that. Thanks. I do agree that adding socket_connect_timeout is unnecessary something should be done about the error. Would it be better to change line in make_connection in connection.py to explicitly pass the arguments?

`return self.connection_class(**self.connection_kwargs)`